hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/AbstractTableLayout.java

```diff
@@ -106,6 +110,10 @@ public void setShowFilterButtonVisible(final boolean visible) {
         tableHeader.setFilterButtonsIconVisible(visible);
     }
 
+    public T getTable() {
```

Comment:
Don't return your internal state so somebody else can refresh the content somehow, some where out of your control.
Better introduce a method `refresh()` which operates on your table. So this class stays in control